code/modules/shuttle/shuttles/dropship.dm

```diff
@@ -148,6 +148,13 @@
 	. = ..()
 	link_landing_lights()
 
+/obj/docking_port/stationary/marine_dropship/Destroy()
+	. = ..()
+	for(var/obj/structure/machinery/landinglight/light in landing_lights)
+		light.linked_port = null
+	landing_lights.Cut()
+	landing_lights = null // We didn't make them, so lets leave them
```

Comment:
FWIW Cut + Nulling are redundant

Reply:
If so then why do the qdel defines do so? https://github.com/cmss13-devs/cmss13/blob/60248f5e5d31b7091e98fabf117cee21634fcc9a/code/__DEFINES/qdel.dm#L52